spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -999,6 +1000,18 @@ def send_app_submission(app_id, version, data)
 
       handle_itc_response(r.body)
 
+      messages = r.body.fetch("messages", {})
+      errors = messages.fetch('error', nil)
+
+      if !errors.nil? and !errors.empty? and errors.first.include?("Problem processing review submission.")
+        if reject_if_waiting_for_review
+          reject_app_submission(app_id, version)
```

Comment:
Did you check that this is actually being called and that it behaves as expected?

Reply:
@minuscorp I have tested the reject app submission option and it's working as intended. Not sure how to fix the test case though.

Reply:
@minuscorp Finally figured out the test case. Fixed it.